Shortening strategies in ASB when using ServiceControl notification events (#5739)
Particular/docs.particular.net: samples/servicecontrol/events-subscription/sample.md

This sample uses the [Learning Transport](/transports/learning/) in order to be portable with no transport dependencies.

> When changing this sample to use Azure Service Bus as messaging infrastructure, note that the subscribing endpoint must not override the default name shortening strategy in [enity creation configuration settings](/transports/azure-service-bus/configuration.md#entity-creation) when using [Azure Service Bus Transport](/transports/azure-service-bus/), nor [sanitization strategy documentation](/transports/azure-service-bus/legacy/sanitization.md) when using [the legacy version of the transport](/transports/azure-service-bus/legacy/). 

Reviewer: How about?

```suggestion
When changing this sample to use Azure Service Bus as a transport, note that the subscribing endpoint must not override the default name shortening strategy. The [entity creation configuration settings](/transports/azure-service-bus/configuration.md#entity-creation) of [Azure Service Bus Transport](/transports/azure-service-bus/) or the [sanitization strategy](/transports/azure-service-bus/legacy/sanitization.md) when using [the legacy version of the transport](/transports/azure-service-bus/legacy/) must not be overriden. 
```
